Testnet/MintableToken/MintableToken/MintableToken.cs

```diff
@@ -221,6 +235,28 @@ public void MintWithMetadata(Address account, UInt256 amount, string metadata)
         Log(new MintMetadata() { To = account, Amount = amount, Metadata = metadata });
     }
 
+    /// <inheritdoc />
+    public void MintWithMetadataForNetwork(Address account, UInt256 amount, string metadata, string destinationAccount, string destinationNetwork)
+    {
+        MintWithMetadata(account, amount, metadata);
+
+        if (string.IsNullOrEmpty(destinationAccount) && ((string.IsNullOrEmpty(destinationNetwork) || destinationNetwork == "CIRRUS")))
```

Comment:
Redundant paranthesis
```suggestion
        if (string.IsNullOrEmpty(destinationAccount) && (string.IsNullOrEmpty(destinationNetwork) || destinationNetwork == "CIRRUS"))
```